cdap-cli/src/main/java/co/cask/cdap/cli/CLIConfig.java

```diff
@@ -201,7 +200,7 @@ private boolean saveAccessToken(AccessToken accessToken, String hostname) {
     File accessTokenFile = getAccessTokenFile(hostname);
 
     try {
-      if (accessTokenFile.createNewFile()) {
+      if (accessTokenFile.exists() || accessTokenFile.createNewFile()) {
```

Comment:
Sounds good. I was also intending to fail silently since saving access token isn't critical, but after reading API docs, looks like the following should be done:

```
try {
  accessTokenFile.createNewFile();
  Files.write(accessToken.getValue(), accessTokenFile, Charsets.UTF_8);
  return true;
} catch (SomeException e) {
  // Ignore
}
```